src/main/java/cubicchunks/lighting/SkyLightCubeDiffuseProcessor.java

```diff
@@ -57,7 +57,7 @@ private void process(Cube cube, long address) {
 
 		Cube.LightUpdateData data = cube.getLightUpdateData();
 
-		Column column = cache.getColumn(columnX, columnZ);
+		Column column = cache.provideChunk(columnX, columnZ);
```

Comment:
Note that `getColumn` does not provide columns in the current (i.e. master) implementation